packages/gestalt/src/Column/Column.js

```diff
@@ -31,10 +31,10 @@ export default function Column(props: ColumnProps) {
     props.sm && styles[`smCol${props.sm}`],
     props.md && styles[`mdCol${props.md}`],
     props.lg && styles[`lgCol${props.lg}`],
-    props.span && styles[`xsCol${props.span}`],
```

Comment:
`Columns` can be literal number `0` which is falsy - previously we wouldn't hit `.xsCol0` classname.